tests/test_responses.py

```diff
@@ -245,6 +246,27 @@ async def app(scope, receive, send):
     assert filled_by_bg_task == "6, 7, 8, 9"
 
 
+def test_file_response_on_head_method(
+    tmpdir: Path, test_client_factory: typing.Callable[..., TestClient]
+):
+    path = os.path.join(tmpdir, "xyz")
+    content = b"<file content>" * 1000
+    with open(path, "wb") as file:
+        file.write(content)
+
+    app = FileResponse(path=path, filename="example.png")
+    client = test_client_factory(app)
```

Comment:
This test is actually useless, since the `TestClient` drops the body on HEAD methods.

I'll create a new test later, but if you want to approve, it will speed up the process here. 👀